docs/home/howto/OS_PATCHING.md

```diff
@@ -56,35 +66,40 @@ Note: For different supported OS versions this guide may be useful as well.
 Azure has `Update Management` solution in `Azure Automation`. It gives you visibility into update compliance across Azure and other clouds, and on-premises. The feature allows you to create scheduled deployments that orchestrate the installation of updates within a defined maintenance window.  
 To manage updates that way please refer to [official documentation](https://docs.microsoft.com/en-us/azure/automation/update-management/update-mgmt-manage-updates-for-vm).
 
-## Patching RHEL with Package Manager
+## Patching with Package Manager
 
-The following commands can be executed in both clustered and non-clustered environments. In case of patching non-clustered environment you have to schedule a maintenance window due to required reboot after kernel patching.  
+The following commands can be executed in both clustered and non-clustered environments. In case of patching non-clustered environment, you have to schedule a maintenance window due to the required reboot after kernel patching.  
 
 Note: Some of the particular patches may also require a system reboot.
 
 If your environment is clustered then hosts should be patched one by one. Before proceeding with the next host be sure that the patched host is up and all its components are running.  
 For information how to check state of specific Epiphany components, see [here](./MAINTENANCE.md).
 
-To verify your current Linux kernel version, use:  
+### RHEL
+
+Verify if *epirepo* is disabled and *official repositories* are enabled:  
+`yum repolist all`
+
+Verify your current Linux kernel version:  
 `sudo uname -r`
```

Comment:
Hmmmm .. let's say I have version 1.2.3 ... what did I just verified? Is that good? Is that bad? What versions are OK?